LRU.py

```diff
@@ -0,0 +1,51 @@
+"""
+https://leetcode.com/problems/lru-cache/
+Design and implement a data structure for Least Recently Used (LRU) cache. It should support the following operations: get and put.
+
+get(key) - Get the value (will always be positive) of the key if the key exists in the cache, otherwise return -1.
+put(key, value) - Set or insert the value if the key is not already present. When the cache reached its capacity, it should invalidate the least recently used item before inserting a new item.
+
+The cache is initialized with a positive capacity.
+
+Follow up:
+Could you do both operations in O(1) time complexity?
+"""
+
+class LRUCache:
+
+    def __init__(self, capacity: int):
+        self.cache = {}
+        self.limit = capacity
+
+    def update(self, key: int, value: int):
+        del [self.cache[key]]
+        self.cache[key] = value
+
+    def get(self, key: int) -> int:
+        if self.cache.get(key):
```

Comment:
this works but you are wasting instructions, `get` gets you a value or a default (in this case None). so do this instead:

```
value = self.cache.get(key)  
if value:
    # do something
    return value
```